test/kubectl-layer.integ.ts

```diff
@@ -14,8 +14,9 @@ const stack = new cdk.Stack(app, 'lambda-layer-kubectl-integ-stack');
 const layer = new KubectlV29Layer(stack, 'KubectlLayer');
 
 const runtimes = [
-  lambda.Runtime.PYTHON_3_7,
+  lambda.Runtime.PYTHON_3_8,
   lambda.Runtime.PYTHON_3_9,
+
```

Comment:
can we add [the rest](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-lambda/lib/runtime.ts#L164-L180) of the runtimes here?

Reply:
added 3.10 and 3.11 for python that were available in the cdk version that is in this project